tests/Unit/inc/Engine/Preload/PartialPreloadSubscriber/preloadAfterAutomaticCachePurge.php

```diff
@@ -4,10 +4,10 @@
 
 use Mockery;
 use Brain\Monkey\Functions;
+use WPMedia\PHPUnit\Unit\TestCase;
```

Comment:
changes in this file don't seem right, was it because of the revert?

Reply:
I think so as there is only one import for this class in the file and it not in the same namespace.